### ISSUE TITLE
Application will not stop on Travis (Windows)

### ISSUE THREAD

**Issue by jkuester** [electron-userland/spectron#370]
I am seeing an issue where running Application.stop() in the [Windows Build Environment](https://docs.travis-ci.com/user/reference/windows/) on Travis just hangs indefinitely every time it is called.   The tests run fine locally on Windows 10, AppVeyor, and also in the Travis Linux and OSX build environments.  
After some debugging I think the cause of the hanging is [this line](https://github.com/electron-userland/spectron/blob/74f87c1d93482072d94c9cb0947d989f1860ebc8/lib/application.js#L76)
```.js
window.close()
```
My tests are running with electron:5.0.5 and spectron7.0.0.

Ultimately, I solved this by using the `no-sandbox` arg when creating a new Application because I noticed [this commit](https://github.com/electron-userland/spectron/commit/1842b4d2a09f0edce717a4a5b09e4d61431dc727) that added AppVeyor support.  So, I guess my question is can anyone tell me if this is the proper solution? If so, should we consider automatically setting the flag when running in the Travis Windows build environment just like we are on AppVeyor?

The TLDR for anyone else experiencing the same issues is that I was able to get around this issue by creating my application with these options: 
```.js
  this.app = new spectron.Application({
    path: electronPath,
    args: [path.join(__dirname, '../../src/index.js')],
    chromeDriverArgs: ['no-sandbox'],
    startTimeout: 50 * 1000,
    quitTimeout: 10 * 1000,
    waitTimeout: 10 * 1000,
  });
```
(Note that the extra-large timeouts were necessary because the Travis Windows build environment seems to run really slow sometimes.)
